presto-ui/src/components/QueryDetail.jsx

```diff
@@ -349,99 +349,75 @@ const HISTOGRAM_PROPERTIES = {
     disableHiddenCheck: true,
 };
 
-class RuntimeStatsList extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            expanded: false
-        };
-    }
+const RuntimeStatsList = ({ stats }) => {
+    const [expanded, setExpanded] = React.useState(false);
```

Comment:
nit: could be just `useState`. but up to you

Reply:
Good catch, missed that one :)